database/dbOperations.go

```diff
@@ -0,0 +1,124 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/LEbauchoir/go_final_project/config"
+	"github.com/LEbauchoir/go_final_project/models"
+)
+
+func NewDbHelper(db *sql.DB) *DbHelper {
+	return &DbHelper{
+		Db: db,
+	}
+}
+
+func (d *DbHelper) AddTask(t models.Task) (string, error) {
+	query := "INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)"
+	res, err := d.Db.Exec(query, t.Date, t.Title, t.Comment, t.Repeat)
+	if err != nil {
+		return "", err
```

Comment:
Лучше оборачивать ошибки везде, чтобы было понятнее откуда она пришла
```suggestion
		return "", fmt.Errorf("error while inserting task %+v: %w", t, err)
```

Reply:
Исправил